.github/workflows/helm-chart.yml

```
@@ -3,70 +3,72 @@ name: Build & publish Helm chart to GHCR
     inputs:
       version:
         description: "Version to tag the images (e.g., 1.0.0)"
-        required: true
-        type: string
+        required: false
+        type: 
 
 env:
-  CHART_PATH: charts/kaneo
+  CHART_PATH: chart
+  /kaneo
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)